server/migrations/20190826135230-insert-threats-and-findings-nomenclatures.js

```diff
@@ -0,0 +1,118 @@
+      { bg: 'Кариери', en: 'Quarries' },
+      { bg: 'Разораване на пасища и мери', en: 'Plowing of pastures' },
+      { bg: 'Зарибяване', en: 'Stocking with fish' },
+      { bg: 'Бракониери', en: 'Poachers' },
+      { bg: 'Залесяване с нетипични видове', en: 'Planting with non-typical species' },
+      { bg: 'Пътища', en: 'Roads' },
+      { bg: 'Мъртъв екземпляр', en: 'Dead individual' },
+      { bg: 'Ветропарк', en: 'Wind park' },
+      { bg: 'Прекомерно обрастване на пасища', en: 'Excessive overgrowth of pastures' },
+      { bg: 'Други', en: 'Other' },
+      { bg: 'Соларен парк', en: 'Solar park' },
+      { bg: 'Инвазивни видове', en: 'Invasive species' },
+      { bg: 'Премахване на храсти', en: 'Removal of shrubs' },
+      { bg: 'Мини ВЕЦ', en: 'Small hydroelectric' },
+      { bg: 'Пресъхващи водни тела', en: 'Dried up waterbodies' },
+      { bg: 'Убит на пътя', en: 'Dead on road' },
+      { bg: 'Ловна преса', en: 'Hunting' },
+      { bg: 'Тровене', en: 'Poisoning' },
+      { bg: 'Безпокойство', en: 'Disturbance' },
+      { bg: 'Иманярство', en: 'Treasure hunting' },
+      { bg: 'Екстремни спортове', en: 'Extreme sports' },
+      { bg: 'Туристическа дейност', en: 'Touristic activities' }
+    ]
+  },
+  {
+    name: 'mammals_findings',
+    values: [
+      { bg: 'Строежи', en: 'Construction / development' },
+      { bg: 'Други', en: 'Other' },
+      { bg: 'Изпражнения', en: 'Feces' },
+      { bg: 'Дупка', en: 'Hole' },
+      { bg: 'Капан', en: 'Trap' },
+      { bg: 'Къртичина', en: 'Molehill' },
+      { bg: 'Стъпка(и)', en: 'Footprint(s)' },
+      { bg: 'Колония', en: 'Colony' }
+    ]
+  },
+  {
+    name: 'herptiles_findings',
+    values: [
+      { bg: 'Строежи', en: 'Construction / development' },
+      { bg: 'Съблекло', en: 'Moult' },
+      { bg: 'Други', en: 'Other' },
+      { bg: 'Изпражнения', en: 'Feces' },
+      { bg: 'Пеещ мъжки', en: 'Singing male' },
+      { bg: 'Дупка', en: 'Hole' },
+      { bg: 'Капан', en: 'Trap' },
+      { bg: 'Къртичина', en: 'Molehill' },
+      { bg: 'Стъпка(и)', en: 'Footprint(s)' },
+      { bg: 'Колония', en: 'Colony' }
+    ]
+  },
+  {
+    name: 'invertebrates_findings',
+    values: [
+      { bg: 'Строежи', en: 'Construction / development' },
+      { bg: 'Съблекло', en: 'Moult' },
+      { bg: 'Други', en: 'Other' },
+      { bg: 'Изпражнения', en: 'Feces' },
+      { bg: 'Пеещ мъжки', en: 'Singing male' },
+      { bg: 'Дупка', en: 'Hole' },
+      { bg: 'Капан', en: 'Trap' },
+      { bg: 'Къртичина', en: 'Molehill' },
+      { bg: 'Колония', en: 'Colony' }
+    ]
+  }
+]
+
+function deleteExistingValues (queryInterface) {
+  return queryInterface.bulkDelete('Nomenclatures', {
+    type: {
+      $in: nomenclatures.map(function (nomenclature) { return nomenclature.name })
+    }
+  })
+}
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    if (queryInterface.sequelize.options.dialect !== 'postgres') return
```

Comment:
why only for pg?

Reply:
because we use hard-coded IDs for the nomenclatures in test fixtures and fixtures cannot be applied